config/feature_flags.yml

```diff
@@ -0,0 +1,6 @@
+# Add feature flags to this file, example:
+# dpl202_enable_angry_cat_feature: When enabled this will add an angry cat to all pages that will try to catch the mouse pointer
```

Comment:
Did you come up with this or is this in the example documentation from Flipper?  Sounds hilarious if someone implemented this.

Reply:
My example. Wanted something that was obviously for demonstration purposes to avoid someone accidentally stripping it out thinking it was a delivered feature.